internal/tracegen/worker_test.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package tracegen
+
+import (
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/jaegertracing/jaeger/pkg/testutils"
+	"github.com/stretchr/testify/assert"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	"go.opentelemetry.io/otel/trace"
+)
+
+func Test_SimulateTraces(t *testing.T) {
+	logger, buf := testutils.NewLogger()
+	tp := sdktrace.NewTracerProvider()
+	tracers := []trace.Tracer{tp.Tracer("stdout")}
+	wg := sync.WaitGroup{}
+	wg.Add(1)
+	var running uint32 = 1
+
+	tests := []struct {
+		name           string
+		worker         *worker
+		expectedOutput string
+	}{
+		{
+			name: "stdout tracer",
+			worker: &worker{
+				logger:  logger,
+				tracers: tracers,
+				wg:      &wg,
+				id:      7,
+				running: &running,
+				Config: Config{
+					Traces:   7,
+					Duration: time.Second,
+					Pause:    time.Second,
+					Service:  "stdout",
+					Debug:    true,
+					Firehose: true,
+				},
+			},
+			expectedOutput: "{\"level\":\"info\",\"msg\":\"Worker 7 generated 7 traces\"}\n",
```

Comment:
you can use back ticks to enclose the string, then you don't have to escape every quote